3.analyze_data/nbconverted/log_reg_class_balanced_areashape_aggregate_well_predicted_probabilities.py

```diff
@@ -0,0 +1,259 @@
+#!/usr/bin/env python
```

Comment:
does this file belong to #23 ?

I recommend merging main into this PR prior to merging

Reply:
Good idea